Sources_v3/Database/DTOs/MessageDTO.swift

```diff
@@ -248,6 +259,7 @@ extension _ChatMessage {
         
         author = dto.user.asModel()
         mentionedUsers = Set(dto.mentionedUsers.map { $0.asModel() })
+        replies = Set(dto.replies.map { $0.asModel() })
```

Comment:
This should load just `X` latest replies - like it's done for `latestMessages` in `Channel`

Reply:
Removed. 
Will be part of the next PR.